adafruit_turtle.py

```diff
@@ -1035,15 +1046,17 @@ def hideturtle(self):
 
     ht = hideturtle
 
-    def isvisible(self):
+    def isvisible(self) -> bool:
         """
         Return True if the Turtle is shown, False if it's hidden."""
         if self._turtle_group:
             return True
         return False
 
     # pylint:disable=too-many-statements, too-many-branches
-    def changeturtle(self, source=None, dimensions=(12, 12)):
+    def changeturtle(
+        self, source: Any = None, dimensions: Tuple[int, int] = (12, 12)
```

Comment:
I think `source` could be a `Union` of `displayio.TileGrid` and `str`,  wrapped with `Optional`.